docs/.vuepress/components/Button.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
I think this component can be simplified to just use one boolean, e.g. `isExternal`. If `isExternal` it true, it uses an `a` tag, otherwise it uses the `router-link` for internal links.